agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/tcs/handler/handler.go

```diff
@@ -54,7 +54,7 @@ type telemetrySession struct {
 	agentHash                     string
 	containerRuntimeVersion       string
 	endpoint                      string
-	disableContainerHealthMetrics bool
+	disableMetrics                bool
```

Comment:
nit: I'm assuming changes here are the result of `go mod vendor` not being run as part of a previous PR. If that is the case, can we update the PR description to explicitly include this (to make it clear that TACS changes are not in scope of this PR, despite this diff being present)?

Reply:
Yeah that's right. I have updated the PR but I will also get the author of the changes to review. Thanks @danehlim!

Reply:
@Realmonia confirmed that the changes are safe. I have updated the PR accordingly.